src/linux_mcp_server/connection/ssh.py

```diff
@@ -127,10 +127,21 @@ async def get_connection(self, host: str, username: str) -> asyncssh.SSHClientCo
         logger.debug(f"{Event.SSH_CONNECTING}: {key} | key={self._ssh_key or 'none'}")
 
         try:
+            # Determine host key verification settings
+            if CONFIG.verify_host_keys:
+                known_hosts = (
+                    str(CONFIG.known_hosts_path)
+                    if CONFIG.known_hosts_path
+                    else str(Path.home() / ".ssh" / "known_hosts")
```

Comment:
This could be handled in the CONFIG class. That way it's only necessary to get the property here.